src/styles/variable.scss

```
@@ -0,0 +1,7 @@
+$fontFamily: Roboto, sans-serif;
+$navLinkColor: #060b35;
+$navColor: #00acdc;
+$buttonColor: #00acdc;
```

```suggestion
$accentColor: #00acdc;
```
or secondryColor

No need to create 2 variables with the same value. Just rename it properly